pages.fr/common/gh.md

```diff
@@ -0,0 +1,36 @@
+# gh
+
+> Travailler harmonieusement avec GitHub depuis la ligne de commande. Certaines sous-commandes telles que `gh config` ont leur propre documentation d'usage.
```

Comment:
We split in 2 lignes the meaning description of the command and the references

```suggestion
> Travailler harmonieusement avec GitHub depuis la ligne de commande.
> Certaines commandes comme `gh config` ont leur propre documentation.
```

Also for `Certaines sous-commandes telles que `gh config` ont leur propre documentation d'usage.` is correct but we use a slightly different translation `Certaines commandes comme `gh config` ont leur propre documentation.` in other files. 

For your next contribution do not hesitate to look at the other files translated to see the convention we have ;)